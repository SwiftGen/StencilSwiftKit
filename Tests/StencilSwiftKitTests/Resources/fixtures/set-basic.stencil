// before set
x = "{{x}}"
y = "{{y}}"

// set value
{% set x %}1{% endset %}
{% set y %}2{% endset %}
x = "{{x}}"
y = "{{y}}"

// set value again
{% set x %}3{% endset %}
{% set y %}{{x}}{% endset %}
x = "{{x}}"
y = "{{y}}"
