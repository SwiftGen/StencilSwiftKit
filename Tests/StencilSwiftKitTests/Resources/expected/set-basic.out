// before set
x = ""
y = ""

// set value
x = "1"
y = "2"

// set value again
x = "3"
y = "3"
